### PR TITLE
add 3rd party license

### PR DESCRIPTION
**Pull request by sharonx**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -198,4 +198,37 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+
+=======================================================================
+Apache Spark Subcomponents:
```
Comment (dbaldwin-splunk) [inline LICENSE]: Splunk Connect for Docker Subcomponents.   Still has Apache Spark

File: LICENSE [hunk 2]
```diff
@@ -198,4 +198,37 @@
...
+
+The Apache Spark project contains subcomponents with separate copyright
```
Comment (dbaldwin-splunk) [inline LICENSE]: Splunk Connect for Docker instead of The Apache Spark project.

Comment (sharonx) [reply LICENSE]: lol my bad

**Review by dbaldwin-splunk — COMMENTED** [review]
Some items remaining from Apache Spark template that need to be removed or changed to reference Splunk Connect for Docker.

**Comment by sharonx** [comment]
Changed!

**Review by luckyj5 — APPROVED** [review]
LGTM. Thanks @sharonx

**Pull request merged**